portfolio/src/main/webapp/test.js

```diff
@@ -0,0 +1,31 @@
+/* 
+* Code snippet for translating console.log to print
+* using regular expressions and mapping between 
+* languages. Only translates console.log statements
+* ignores all other statements. 
+*/
+
+let userCode = "console.log(x);\n"; 
+userCode += "   console.log(1 + 2);\n";
+userCode += " 1 * 5;\n"
+userCode += "console.log(\"simple string\");\n"
+userCode += " console.log(\"hello\" + \"world\")"; 
+
+// matches all console.log statements
+const consoleLogPattern = /console\.log\((.*)\)/g;
```

Comment:
Here's (definitely an edge case) that might cause problems here.

Very awkward for them to put this all on one line but it might cause problems.
`(() => { console.log("Hello World") })();`

Not against pushing this edge case to a future PR though - just add a comment for now.